frontend/src/metabase/visualizations/echarts/cartesian/model/series.ts

```diff
@@ -634,14 +636,18 @@ const createSeriesLabelsFormatter = (
 ) =>
   cachedFormatter((value: RowValue) => {
     if (typeof value !== "number") {
-      return " ";
```

Comment:
I think there may have been a reason we did `" "` instead of `""`, I remember Sasha wrote a comment about it but I can't find it now. To be safe maybe we should just keep it as a space.

Reply:
I actually created an ECharts patch on a prior PR ([#42985](https://github.com/metabase/metabase/pull/42985)) that was specifically addressing how label are handled by ECharts.
If we keep the `" "` instead of a `""`, it will cause non-empty labels to potentially be hidden because `" "` still has a non-zero length and hence non-zero width and height.